samples/wifi/shutdown/README.rst

```diff
@@ -57,6 +57,20 @@ The following is an example of the CLI command to demonstrate Wi-Fi shutdown:
 
    west build -b nrf7002dk_nrf5340_cpuapp
 
+Disable auto-start of the Wi-Fi driver
+--------------------------------------
+
+The Wi-Fi network interface is automatically brought up when the Wi-Fi driver is initialized by default.
+But you can disable this behavior by setting the ``CONFIG_WIFI_INIT_AUTO_START`` configuration option to ``n``.
+
+Please use additional configuration file ``overlay-no_wifi_auto_start.conf`` to disable auto-start of the Wi-Fi driver:
+
+.. code-block:: console
+
+   west build -b nrf7002dk_nrf5340_cpuapp -DEXTRA_CONF_FILE=overlay-no_wifi_auto_start.conf
+
+With this configuration, the Wi-Fi network interface is not automatically brought up by the sample.
```

Comment:
```suggestion
With this configuration, the Wi-Fi network interface is not automatically brought up by the nRF70 driver initialization.
```

Reply:
Well, its the networking stack that brings up the interface, so, `networking stack`